lib/dbBackupRecovery.js

```diff
@@ -1,74 +1,83 @@
-"use strict";
-var fs = require('fs');
-const path = require('path');
-const {OpenShiftClient} = require('@bcgov/pipeline-cli');
-const config = require(`${process.cwd()}/lib/config.js`);
-const {childProcess} = require('./util-functions');
-const CONST = require('./constants');
-
-(async function () {
-
-    const env = config.options.env;
-    const op = config.options.op;
-    const tnsAdminSecretName = config.phases.dev.backupRecovery.tnsAdminSecretName;
-    const dbCredentialSecretName = config.phases.dev.backupRecovery.dbCredentialSecretName;
-
-    await backupRecovery(env, op);
+'use strict'
+const fs = require('fs')
+const path = require('path')
+// TOFIX: never used?
```

Comment:
This file is not used yet. and i dont think we need pipeline-cli in this file either. So yes.. it is good to be commented out